sdk/highlight-next/src/index.edge.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiHandler, HasHeaders, HasStatus } from './util/withHighlight'
+
+import { NodeOptions } from '@highlight-run/node'
+
+export function Highlight(_: NodeOptions) {
```

Comment:
We do not currently support the `edge` runtime for Next.js. This is an attempt to implement a noop for `edge` so it doesn't throw an error.